Java/syntax_test_java.java

```diff
@@ -1314,9 +1574,11 @@ public V apply(V input) {
 // <- storage.type.java
 
 public class Generic<T> implements fully.qualified.Other<T> {
```

Comment:
fully.qualified.Other should be in `meta.path`.

Reply:
It is, it's the test that is missing.

Reply:
Added test.